test/unit/mock/profiles/supported_inspec/inspec.yml

```diff
@@ -1,2 +1 @@
-supports:
-- inspec: 0.18
+inspec_version: 0.18
```

Comment:
What was the reason behind this change? I cannot remember.

Reply:
We wanted supports to only have platform conditions in it. So we didn't have a mixture.

Reply:
@arlimus Is that matching your expectations?

Reply:
I'd like to add a 👍  to this change to make is super-clear. Especially since we don't want resource authors to add an inspec version check to the `supports` inside a resource definition itself, because that would make zero sense. Separating it and isolating it to the `inspec.yml` is cleaner, IMO.

Reply:
I'm 👍 on changing this here.

The resource  cross-dependency issue looks broad and should be tackled separately